### PR TITLE
Windows Reproducible Script - Fix Path Length

### PR DESCRIPTION
**Pull request by steelhead31**
Amend the working directory to be a significantly shorter path, and copy the output diff file to a more usable location.

### CODE REVIEW

**Comment by steelhead31** [comment]
See #3711

**Review by andrew-m-leonard — APPROVED** [review]

File: tooling/reproducible/windows_repro_build_compare.sh [hunk 1]
```diff
@@ -39,8 +39,8 @@ TARBALL_URL="$2"
 # These Values Should Be Updated To Reflect The Build Environment
 # The Defaults Below Are Suitable For An Adoptium Windows Build Environment
 # Which Has Been Created Via The Ansible Infrastructure Playbooks
-CURR_DIR=$(pwd)
-WORK_DIR="$CURR_DIR/cmp$(date +%Y%m%d%H%M%S)"
+
+WORK_DIR="C:/workspace/comp-jdk-build/"
```
Comment (sophia-guo) [inline tooling/reproducible/windows_repro_build_compare.sh:43]: Would this dir be available for dev's local environment?

Comment (steelhead31) [reply tooling/reproducible/windows_repro_build_compare.sh:43]: Yes, the script creates it, if it doesn't exist...

Comment (steelhead31) [reply tooling/reproducible/windows_repro_build_compare.sh:43]: Ah, but I can probably make this better...

Comment (steelhead31) [reply tooling/reproducible/windows_repro_build_compare.sh:43]: Let me make a quick fix..

Comment (steelhead31) [reply tooling/reproducible/windows_repro_build_compare.sh]: should be ok now.

**Comment by sophia-guo** [comment]
Can you also add move or add the compare results (.diff ) to current dir as the result dir, which can be accessed by aqa-tests?

**Comment by steelhead31** [comment]
> Can you also add move or add the compare results (.diff ) to current dir as the result dir, which can be accessed by aqa-tests?

It currently puts it in  WORK\_DIR as seen below,  I'll see if I can move it to the directory the script was launched from..   
  
`mv "$WORK_DIR/compare/repro_diff.out" "$WORK_DIR"`

**Comment by steelhead31** [comment]
@sophia-guo it will now record the directory win\_repro.sh script and copy the repro\_diff.out file to the same location, ie..   
  
`ScriptPath="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"`  
`cp "$WORK_DIR/compare/repro_diff.out" "$ScriptPath"`  
 

Please let me know if that will work for you?

**Comment by steelhead31** [comment]
Found another bug, the script wasnt returning from its call to the repro\_compare.sh script, so I've added a fix for that, which Im just testing.

**Review by sophia-guo — APPROVED** [review]

**Pull request merged**